packages/gateway-client/src/worker/p2p-fetch/p2p-fetch-request.ts

```diff
@@ -1,287 +1,29 @@
-import { Buffer } from 'buffer';
-
 import { logger } from '../logging';
 import { P2pClient } from '../p2p-client';
-import { encode, Packet } from './lob-enc';
-import { P2pRequest } from './p2p-request';
-import transformers from '../transformers';
-
-type PatchedBody = ReturnType<P2pFetchRequest['patchArgs']>['body'];
-type PatchedRequest = ReturnType<P2pFetchRequest['patchArgs']>['reqObj'];
-type PatchedRequestInit = ReturnType<P2pFetchRequest['patchArgs']>['reqInit'];
 
 export class P2pFetchRequest {
-    private log = logger.getLogger('worker/p2p-fetch/fetch-request');
-
-    private chunkSize = 1024 * 64;
-
-    private requestId = crypto.randomUUID();
-
-    public body: PatchedBody;
-    public reqObj: PatchedRequest;
-    public reqInit: PatchedRequestInit;
-
-    private givenReqObj: Request;
+    private request: Request;
 
     constructor(
         private p2pClient: P2pClient,
-        givenReqObj: URL | RequestInfo,
-        givenReqInit: RequestInit | undefined = {}
+        givenReqObj: Request | URL | string,
+        _givenReqInit: RequestInit | undefined = {}
     ) {
         // assert that we were given a request and store it
-        this.givenReqObj = givenReqObj = givenReqObj as Request;
+        this.request = givenReqObj = givenReqObj as Request;
```

Comment:
Let's just move everything in this class into the `p2pFetch()` method and get rid of this file as well.